task_processor/apps.py

```diff
@@ -0,0 +1,20 @@
+from django.apps import AppConfig
+from django.conf import settings
+from health_check.plugins import plugin_dir
+
+from task_processor.task_run_method import TaskRunMethod
+
+
+class TaskProcessorAppConfig(AppConfig):
+    name = "task_processor"
+
+    def ready(self):
+        from . import tasks  # noqa
```

Comment:
It doesn't look as if this is getting used anywhere? If there is a side effect by importing it I'd add a comment explaining that.

Reply:
Yeah, this is to ensure that the module is loaded, and hence the tasks are registered. I've added a comment.